fixUrls.js

```diff
@@ -29,8 +29,9 @@ module.exports = function (css) {
   var currentDir = baseUrl + location.pathname.replace(/\/[^\/]*$/, "/");
 
 	// convert each url(...)
-	var fixedCss = css.replace(/url *\( *(.+?) *\)/g, function(fullMatch, origUrl) {
+	var fixedCss = css.replace(/url\(((?:[^)(]+|\((?:[^)(]+|\([^)(]*\))*\))*)\)/g, function(fullMatch, origUrl) {
```

Comment:
Would it be acceptable to bring in a dependency instead? This regex is only necessary because standard regex in the browser doesn't support recursive matching.

https://www.npmjs.com/package/xregexp  could do this in a cleaner manner.  

I'll try to explain it in a comment but its basically just a nasty way to recursively match brackets